### DISCUSSION TITLE
Audio jack replacement?

### DISCUSSION THREAD

**Discussion by sabaker** [cardonabits/haxo-hw#92] [Q&A]
Sorry to say it broke off and I promptly lost it.  Can someone please point me at the appropriate replacement part? Perhaps at Adafruit or Jameco?

**Comment by jcard0na** [comment]
Hi @sabaker 

Sorry to hear that!  Please check this post for alternative parts:  https://github.com/cardonabits/haxo-hw/discussions/89#discussioncomment-10180304

Cheers,
